### ISSUE TITLE
[Improvement] Add a Rotate Function in the image viewer

### ISSUE THREAD

**Issue by fux0r2009** [leonlatsch/Photok#54]
When I add most images to Photok they are not imported in the right position (vertical photos imported as horizontal and there is no way to rotate the image inside Photok)

It would be good to keep the imported position and also to offer a rotation function inside the image viewer to rotate the photos.

**Comment by leonlatsch** [comment]
The wrong rotation is a bug that I have encoutered a while back even before release.

So this would be a seperate bug issue. (I will create one)

**Comment by fux0r2009** [comment]
> The wrong rotation is a bug that I have encountered a while back even before release.
> 
> So this would be a separate bug issue. (I will create one)

Yes but even this bug as long as the user can rotate the images and it is remembered by photok to be kept in that orientation it's still better than no orientation at all, looking forward to it!

**Comment by leonlatsch** [comment]
@fux0r2009 If I add this feature, should the roation be saved?
It would be easy to add a simple rotate button, just to view it in another way, but saving this, is something bigger.

**Comment by fux0r2009** [comment]
@leonlatsch Well at least if you can add the rotation button that's what's important, saving the orientation is just an nice to have... So please make the rotate button and if saving option is difficult and time consuming than we can do without :)

**Comment by leonlatsch** [comment]
@fux0r2009 Okay, I will add the button and do some testing with saving it. If it turns out to be complicated, I won't do it.

**Comment by leonlatsch** [comment]
**Final Definition**
- Add a Rotate Button in the image viewer
- Try to save rotation, of its not that complicated

**Comment by fux0r2009** [comment]
@leonlatsch Yes thank you very much that would be more than enough and as always excellent work :) 👍
